pkg/controllers/repo_config.go

```diff
@@ -68,6 +68,12 @@ func RepoConfig() (models.MultiRepoConfig, Error) {
 		return repoConfig, Error{}
 	} else if utils.Exists(ymlRepoConfigFile) {
 		utils.LogWarning(fmt.Sprintf("Found %s file, please rename to %s for repo configuration", ymlRepoConfigFile, repoConfigFileName))
+	} else {
+		// If no config file exists, then this is for an interactive setup, so
+		// return a MultiRepoConfig object containing an empty ProjectConfig object
+		var repoConfig models.MultiRepoConfig
+		repoConfig.Setup = append(repoConfig.Setup, models.ProjectConfig{})
+		return repoConfig, Error{}
```

Comment:
Just a note, but from a Go standards perspective I don't think I like what we are doing with this `Error` object. But I get that it would require a greater refactor then the scope of this issue. Something I will look into when I have cycles for a redesign. 

Other then that I will approve the Go code, but recommend you also wait for an approval from someone with a deeper understanding of the business logic.